### PR TITLE
Person details

### PR DESCRIPTION
**Pull request by DavidColemanUK**
Next stage of Person Details added.

### CODE REVIEW

File: RollOfHonour/src/RollOfHonour.Core/Models/Decoration.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace RollOfHonour.Core.Models;
 
-public class Decoration
+public record Decoration
```
Comment (bashout) [inline RollOfHonour/src/RollOfHonour.Core/Models/Decoration.cs:3]: Nice, records are interesting

File: RollOfHonour/src/RollOfHonour.Core/Models/Person.cs [hunk 2]
```diff
@@ -1,17 +1,88 @@
...
 public class Person
 {
   public int Id { get; set; }
   public string FirstNames { get; set; } = string.Empty;
   public string Initials { get; set; } = string.Empty;
   public string LastName { get; set; } = string.Empty;
-  public string Rank { get; set; } = string.Empty;//TODO: This needs cleaning up - so many similar entries. Autopicker/Prompt perhaps
+
+  public string Rank { get; set; } =
+    string.Empty; //TODO: This needs cleaning up - so many similar entries. Autopicker/Prompt perhaps
```
Comment (bashout) [inline RollOfHonour/src/RollOfHonour.Core/Models/Person.cs:13]: We'll definitely need to standardise this list, yes.

File: RollOfHonour/src/RollOfHonour.Web/Pages/Person/Details.cshtml [hunk 3]
```diff
@@ -4,62 +4,250 @@
...
+            @*                             <a href="/images/gallery/example1.jpeg" title="Photo of Reginald Dudley Bawdwen Anderson "> *@
+            @*                                 <div class="picture-block short styled"> *@
+            @*                                     <div class="picture-border"> *@
+            @*                                         <div class="picture-embed" style=" *@
+            @* 												background-image: url(/images/gallery/example1.jpeg); *@
+            @* 											  "></div> *@
+            @*                                     </div> *@
+            @*                                 </div> *@
+            @*                             </a> *@
+            @*                         </div> *@
```
Comment (bashout) [inline RollOfHonour/src/RollOfHonour.Web/Pages/Person/Details.cshtml:168]: Is this left in on purpose?

**Review by bashout — CHANGES_REQUESTED** [review]
Just a question about that commenting out markup. That's all. Otherwise looks good.

File: RollOfHonour/src/RollOfHonour.Web/Pages/Person/Details.cshtml (hunk 3 above)
Comment (DavidColemanUK) [reply RollOfHonour/src/RollOfHonour.Web/Pages/Person/Details.cshtml:168]: Yes, the photographs section, not implemented yet.

File: RollOfHonour/src/RollOfHonour.Core/Models/Person.cs (hunk 2 above)
Comment (DavidColemanUK) [reply RollOfHonour/src/RollOfHonour.Core/Models/Person.cs:13]: Formatted code. XD

File: RollOfHonour/src/RollOfHonour.Core/Models/Decoration.cs (hunk 1 above)
Comment (DavidColemanUK) [reply RollOfHonour/src/RollOfHonour.Core/Models/Decoration.cs:3]: Don't need to go back and change all the others at this point, but it felt relevant enough to use there. :)

**Review by bashout — APPROVED** [review]
Looks good.

**Pull request merged**